tests/run/i13252.scala

```diff
@@ -0,0 +1,12 @@
+object Test:
```

Comment:
Consider naming this file `i13252a.scala` and the directory below `i13252b`. Having two separate tests with the same name, only differing by one being a file and the other a directory, is confusing.

Reply:
God idea.